### DISCUSSION TITLE
Complete the Friv library

### DISCUSSION THREAD

**Discussion by kpro199procreator** [Tsyuay/FrivReborn#28] [Q&A] [answered]
Hello Tsyuay, I would like to help you complete all the missing games in the friv library, since on many occasions I have come across people who want a certain game that is in the menu and when clicking on it, the 404 error appears

Greetings and I look forward to your prompt response. xd

I have a little experience creating html pages and I managed to create half of friv

I hope you accept my help

**Comment by Tsyuay** [comment, marked as answer]
Hi, Completing the Friv library is still a work in progress, and thinking of adding a contributor would be nice, if you are expert in adding games to the library, we will add you, since we don't want anyone to mess up the repository. Regards, Panda Fake Projects.
![133683071-2872abde-b453-4eb5-8a52-852eea0ea6bb](https://user-images.githubusercontent.com/83605971/211408132-46694bb7-9ec6-41d5-a206-7eb86824805f.png)

**Reply by kpro199procreator** [reply]
hello again tsyuay I have a little experience with html and swf games, my intention is not to ruin your repository, on the contrary I want to support the project so that it is completed. I am willing to help you in any way I can and
    Thank you for reading


greetings from Mexico

**Reply by kpro199procreator** [reply]
I worked with the friv library but those files were lost on a laptop that I no longer have

**Reply by Tsyuay** [reply]
Done :), https://github.com/Tsyuay/FrivReborn/invitations you can go to this link if you don't get an invite.

**Reply by kpro199procreator** [reply]
I'll start as soon as I can

**Reply by kpro199procreator** [reply]
I have already finished a game with its other 2 variants
![e115ce49e7a1ddf315cfef76e8a8d00c](https://user-images.githubusercontent.com/107533003/211965018-2fd923a2-bdab-4258-873f-a97fb14f6f9c.png)
